packages/react-components/react-drawer/stories/src/Drawer/DrawerMultipleLevels.stories.tsx

```diff
@@ -53,44 +55,25 @@ const useStyles = makeStyles({
 });
 
 const BodyPresenceMotion = createPresenceComponent<{ level: 1 | 2 }>(({ level }) => {
```

Comment:
This could almost be a `Slide` variant, except for the slide distance depending on `level` at runtime.